src/components/UserSignUp.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './component.css';
+import { Button, InputGroup, Input } from 'reactstrap';
+
+function UserSignup() {
+  const [FirstName, setFirstName] = useState('');
```

Comment:
@sayalisheth use pascal case for variable name.

Reply:
Done